lib/js_render/renderer.rb

```diff
@@ -12,8 +12,12 @@ class Renderer
       var global = global || this;
       var self = self || this;
       var window = window || this;
+      var console = console || { history: [] };
     JS
 
+    CONSOLE_POLYFILL = File.read(File.join(File.dirname(__FILE__), 'console_polyfill.js'))
+    CONSOLE_REPLAY = File.read(File.join(File.dirname(__FILE__), 'console_replay.js'))
```

Comment:
can we move these into a scripts folder in the same directory? I think that will help organize if we add more polyfills in the future.

Reply:
Good idea 👍